populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json

```diff
@@ -48,6 +48,26 @@
       ],
       "currentPageNo": 1,
       "complete": true,
+      "submittedHoursAgo": 247
+    }, {
+      "surveyStableId": "hd_hd_lifestyle",
+      "surveyVersion": 1,
+      "answerPopDtos": [
+        { "questionStableId": "hd_hd_lifestyle_daysVigorousActivity", "stringValue": "2days"},
```

Comment:
I think this should be `2Days`, right now it doesn't display properly in the preview